### ISSUE TITLE
Build failure of latest git on Debian

### ISSUE THREAD

**Issue by darrenfreeman** [zfsonlinux/zfs#7067]
## System information

Type                                | Version/Name
  ---                                  |     --- 
Distribution Name       | Debian
Distribution Version    | 9.3
Linux Kernel                 | 4.9.0-5-amd64 #1 SMP Debian 4.9.65-3+deb9u2
Architecture                 | x86-64
ZFS Version                  | latest git
SPL Version                  | latest git

Fails to build when I run "make pkg".

```
Making all in libzfs
make[8]: Entering directory '/tmp/zfs-build-root-pvy7frGe/BUILD/zfs-0.7.0/tests/zfs-tests/tests/functional/libzfs'
  CC       many_fds.o
  CCLD     many_fds
../../../../../lib/libzfs/.libs/libzfs.so: undefined reference to `lzc_reopen'
../../../../../lib/libzfs/.libs/libzfs.so: undefined reference to `lzc_load_key'
../../../../../lib/libzfs/.libs/libzfs.so: undefined reference to `lzc_unload_key'
../../../../../lib/libzfs/.libs/libzfs.so: undefined reference to `lzc_change_key'
```

**Comment by darrenfreeman** [comment]
I should note that the release tarball for 0.7.5 builds correctly. But I wanted encryption, so repeating the now familiar procedure on the latest git.

**Comment by phreaker0** [comment]
seems to be a duplicate of my issue #6539

**Comment by gmelikov** [comment]
Closed as duplicate of #6539.
